src/test/java/org/opensearch/neuralsearch/processor/combination/RRFScoreCombinationTechniqueTests.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.combination;
+
+import java.util.List;
+import java.util.Map;
+
+public class RRFScoreCombinationTechniqueTests extends BaseScoreCombinationTechniqueTests {
+
+    private ScoreCombinationUtil scoreCombinationUtil = new ScoreCombinationUtil();
+
+    public RRFScoreCombinationTechniqueTests() {
+        this.expectedScoreFunction = (scores, weights) -> RRF(scores, weights);
+    }
+
+    public void testLogic_whenAllScoresPresentAndNoWeights_thenCorrectScores() {
+        ScoreCombinationTechnique technique = new RRFScoreCombinationTechnique(Map.of(), scoreCombinationUtil);
+        testLogic_whenAllScoresPresentAndNoWeights_thenCorrectScores(technique);
+    }
+
+    public void testLogic_whenNotAllScoresPresentAndNoWeights_thenCorrectScores() {
+        ScoreCombinationTechnique technique = new RRFScoreCombinationTechnique(Map.of(), scoreCombinationUtil);
+        testLogic_whenNotAllScoresPresentAndNoWeights_thenCorrectScores(technique);
+    }
+
+    private float RRF(List<Float> scores, List<Double> weights) {
+        float sumScores = 0.0f;
+        for (float score : scores) {
+            sumScores += score;
+        }
+        return sumScores;
+    }
```

Comment:
Why are you adding these method in this testing? I think you can simply with a few examples like 1 plus 1 is 2.

Reply:
it's added to be compatible with https://github.com/opensearch-project/neural-search/blob/main/src/test/java/org/opensearch/neuralsearch/processor/combination/BaseScoreCombinationTechniqueTests.java and be able to use all test cases it provides. We need to ensure in better possible test coverage if it's a low hanging fruit

Reply:
Did not get your point. The private `randomScore` outputs non-deterministic results.